pkg/kubelet/cadvisor.go

```diff
@@ -24,6 +25,17 @@ import (
 	cadvisor "github.com/google/cadvisor/info"
 )
 
+var (
+	// ErrNoKubeletContainers returned when there are not containers managed by the kubelet (ie: either no containers on the node, or none that the kubelet cares about).
+	ErrNoKubeletContainers = errors.New("No containers managed by kubelet")
+
+	// ErrContainerNotFound returned when a container in the given pod with the given container name was not found, amongst those managed by the kubelet.
+	ErrContainerNotFound = errors.New("No matching container")
+
+	// ErrContainerInfoFailed returned when cadvisor couldn't retrieve stats for the given container, either because it isn't running or it was confused by the request
+	ErrContainerInfoFailed = errors.New("Failed to retrieve cadvisor stats")
```

Comment:
nit: How about `ErrCadvisorApi` instead? ContainerInfo is specific to cadvisor's API and doesn't make sense outside of this file.